### PR TITLE
Add a DatasetScan for the 2019 Blizzard satellite data

### PR DESCRIPTION
**Pull request by dopplershift**

### CODE REVIEW

**Comment by dopplershift** [comment]
Requesting a review since this is the live server we're talking about.

File: thredds/casestudies/catalog.xml [hunk 1]
```diff
@@ -86,7 +86,19 @@
     </dataset>
 
 
-
+  <dataset name="2019 March Plains Blizzard">
+    <datasetScan name="GOES Imagery" ID="casestudies/2019blizzard/goes"
+                 path="casestudies/2019blizzard/goes",
```
Comment (lesserwhirls) [inline thredds/casestudies/catalog.xml]: The trailing comma will lead to a bad time at startup.

**Comment by lesserwhirls** [comment]
Other than the rogue comma, looks good to me. 👍

**Comment by lesserwhirls** [comment]
I made a test using `pytest` that will at least check to make sure all xml files in the repo are well-formed. Maybe we could add it to this repo together (along with all the travis stuff we'd need)?

**Comment by dopplershift** [comment]
Sounds great. Would really love an automated way to avoid server startup failure.

**Pull request merged**